DuggaSys/diagram.js

```diff
@@ -820,8 +820,6 @@ var movingContainer = false;
 //Grid Settings
 var settings = {
     ruler: {
-        lineRatio: 10,
```

Comment:
This can still be used to set a const lineRatio as you did at row 3673, so it does not HAVE to be removed.

example: const lineRatio = settings.ruler.lineRatio;

Not sure if it makes a difference unless we want to keep all grid settings within "settings".